src/common/util.c

```diff
@@ -442,3 +442,35 @@ int oscap_open_writable(const char *filename)
 	}
 	return fd;
 }
+
+bool oscap_path_startswith(const char *path, const char *prefix)
```

Comment:
This function is a pivoting point of the path blocking. And I would really like its behavior to be documented and validated in a form of a unit test (absolute vs canonical path, empty, invalid etc).

A good place for such a test is `tests/common/test_oscap_util.c` module.